lib/statsd.js

```diff
@@ -62,6 +62,15 @@ function makeStatsD(options, logger) {
         statsd = new StatsD(statsdOptions);
     }
 
+    // Support creating sub-metric clients with a fixed prefix. This is useful
+    // for systematically categorizing metrics per-request, by using a
+    // specific logger.
+    statsd.makeChild = function(name) {
+        var childOptions = Object.assign({}, options);
+        childOptions._prefix = statsdOptions.prefix + normalizeName(name) + '.';
```

Comment:
The prefix is set at https://github.com/gwicke/service-runner/blob/makeChild/lib/statsd.js#L40, which will only work if it's a string.